src/strings/index.tsx

```diff
@@ -940,6 +940,8 @@ const strings = new LocalizedStrings({
     UNSPECIFIED: 'Unspecified',
     UNSURE: 'Unsure',
     UPDATE_STATUS_WARNING: 'You‘re about to update the status',
+    UPDATED_TIMEZONE_TITLE: 'Time zone was updated to {0} zone.',
```

Comment:
`Your time zone..`?
In case we need to differentiate between user and org at some point